pagination/helpers.go

```diff
@@ -3,16 +3,31 @@ package pagination
 import (
 	"encoding"
 	"encoding/base64"
+	"errors"
+	"github.com/gazebo-web/gz-go/v7/repository"
+	"github.com/gazebo-web/gz-go/v7/repository/firestore"
 	"time"
 )
 
-// PageSizeGetter holds a method to return the amount of pages requested by users when listing items in an API call.
-type PageSizeGetter interface {
+// Pagination determines the methods needed in a request object to perform pagination.
+type Pagination interface {
+	pageSizeGetter
+	pageTokenGetter
+}
+
+// pageSizeGetter holds a method to return the amount of pages requested by users when listing items in an API call.
+type pageSizeGetter interface {
```

Comment:
This is used by the public function `GetListAndCursor`. Let's make all interfaces in this file public in case we choose to add additional functions using them in the future.